JavaAssignments/src/rahulV/Assignment3_Q1.java

```diff
@@ -0,0 +1,34 @@
+package rahulV;
+
+import java.util.Scanner;
+
+public class Assignment3_Q1 {
+
+	void largestNum() { // method to display largest number
```

Comment:
Ravi, Its always better to create function with parameters. take input from main method and method should accept 3 parameters. so other class can also use your function on their computed values (consider a case where other class want to find largest number but dont want to ask user for values)